Lib/random.py

```diff
@@ -67,7 +68,7 @@
 SG_MAGICCONST = 1.0 + _log(4.5)
 BPF = 53        # Number of bits in a float
 RECIP_BPF = 2**-BPF
-
+_ONE = 1
```

Comment:
Why would you do this? This seems like bad coding style.  Nothing better communicates the constant one than the constant one.

Reply:
To distinguish the default value for `step` from other values which are equal to 1 (e.g. 1.0, Fraction(1, 1), Decimal('1'), etc).

There are two issues with using just
```python
    if step is 1:
```
1. It is not guaranteed that 1 is a singleton. It is a CPython implementation detail.
2. For this reason this code emits a compiler warning.

Reply:
Please just use the constant 1 and use an equality test instead of identity test.